Rakefile

```diff
@@ -9,3 +10,36 @@ PuppetLint.configuration.send('disable_class_inherits_from_params_class')
 PuppetLint.configuration.send('disable_documentation')
 PuppetLint.configuration.send('disable_single_quote_string_with_variables')
 PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "pkg/**/*.pp"]
+
+default_acceptance_platform = 'windows2012r2-64'
+
+desc 'Generate Beaker Host config'
+rototiller_task :host_config, [:default_platform] do |t, args|
+  t.add_env({:name => 'PLATFORM', :message => 'PLATFORM Must be set. For example "windows2012r2-64"', :default => args[:default_platform]})
+  hosts_file = "spec/acceptance/nodesets/#{ENV['PLATFORM']}.yml"
+  t.add_command do |cmd|
+    cmd.name = 'bundle exec beaker-hostgenerator'
+    cmd.add_argument({:name => "#{ENV['PLATFORM']} > #{hosts_file}"})
+  end
+end
+
+desc 'Executes acceptance tests'
+rototiller_task :acceptance do |t|
```

Comment:
The puppetlabs_spec_helper library provides a `beaker` task doing almost the same, and a few other utility tasks (see https://github.com/puppetlabs/puppetlabs_spec_helper/blob/2f22889b8ce15caa0545bac271fdaced77352caa/lib/puppetlabs_spec_helper/rake_tasks.rb#L36 and L371 ff).